.buildkite/release-automation/pre_release.rayci.yml

```diff
@@ -0,0 +1,36 @@
+group: Pre-release checks
+steps:
+  - label: "Check release blockers"
+    key: check-release-blockers
+    depends_on: forge
+    job_env: forge
+    instance_type: small_branch
+    commands:
+      - bash .buildkite/release-automation/check_release_blockers.sh
+
+  - block: "Trigger tests & image builds"
+    depends_on:
+      - step: check-release-blockers
+        allow_failure: true
+
+  - label: "Trigger Postmerge test"
+    trigger: "postmerge"
+    if: build.env("COMMIT") != null && build.env("BRANCH") != null
+    build:
+      commit: "${COMMIT}"
+      branch: "${BRANCH}"
+      message: "Triggered by release-automation build #${BUILDKITE_BUILD_NUMBER}"
+      env:
+        NIGHTLY: 1
```

Comment:
what 'nightly: 1' does?

also a todo, are the non-flaky mac os tests blocking yet?

Reply:
My idea is to add `NIGHTLY=1` as an environment variable in every scheduled nightly build, and then enforce that we only build images and tag `nightly` when `NIGHTLY=1` which only happens on scheduled nightly runs. It's used in this PR: https://github.com/ray-project/ray/pull/43486

Reply:
got you

Reply:
^ just to clarify, we still build images and push with the commit tag normally, just not have `nightly` alias when `NIGHTLY != 1` or `NIGHTLY` doesn't exist in the build env

Reply:
let's maybe remove the envs for now. and leave a todo mark here, and we backfill this env thing in another PR?